ui/src/app/shared/components/checkbox-filter/checkbox-filter.scss

```diff
@@ -0,0 +1,20 @@
+@import 'node_modules/argo-ui/src/styles/config';
+
+.checkbox__filter {
```

Comment:
Nitpick:

The `__` is used as a separate between block and element name in BEM notation. So class names should be: `checkbox-filter` and `checkbox-filter__label`.

Reply:
Thanks. done.